kernel/src/fs/path/mount.rs

```diff
@@ -66,6 +158,8 @@ pub struct Mount {
     mnt_ns: Weak<MountNamespace>,
     /// Propagation type of this mount (e.g., private, shared).
     propagation: RwLock<MountPropType>,
+    /// The flags of this mount.
+    flags: Mutex<PerMountFlags>,
```

Comment:
I think that eventually, we will need an atomic version of `PerMountFlags`. Such a flag will be consulted frequently during certain FS operations when we fully respect these flags in the future. In fact, I think both `PerMountFlags` and `FsFlags` need to have the corresponding atomic versions to make these flags cheaply accessible and updatable.